app/Event/EventGeoCoordinatesServiceProvider.php

```diff
@@ -36,6 +36,8 @@ function () use ($container): GeoCoordinatesCommandHandler {
                     new FullAddressFormatter(),
                     new LocalityAddressFormatter(),
                     $container->get(GeocodingService::class),
+                    $container->get('event_jsonld_repository'),
+                    $container->get('config')['add_location_name_to_coordinates_lookup'] ?? false
```

Comment:
Don't forget to create a pull request on https://github.com/cultuurnet/appconfig and set `add_location_name_to_coordinates_lookup` to true for ACC, but still to false for TEST and PROD

Reply:
See PR https://github.com/cultuurnet/appconfig/pull/518